docs/src/content/pages/installation-astro.mdoc

```diff
@@ -193,7 +193,67 @@ Notice we also opt out of prerendering for those routes.
 
 We should be all set to use Keystatic now.
 
-### Creating a new post
+## Disabling indexing
```

Comment:
```suggestion
## Preventing indexing of Keystatic routes in production
```

Reply:
I think it's worth being more specific here to avoid confusion:

```suggestion
## Preventing indexing of Keystatic Admin UI routes in production
```